src/kruskal.jl

```diff
@@ -0,0 +1,52 @@
+####################################################
+## Minimum Spanning Tree - Kruskal algorithm
+####################################################
+
+"""
+
+Computes the minimum spanning tree of an undirected graph by using Kruskal's algorithm.
+
+Requires arguments:
+distmx::LightGraphs.Graph          # Adjacency matrix of the input graph
```

Comment:
I'm not sure what this means. distmx is generally a matrix (2x2), not a Graph. Also, we don't need to prefix with LightGraphs here, do we?